### ISSUE TITLE
bindings.azure.storagequeues don't support encodeBase64 when send message

### ISSUE THREAD

**Issue by heavenwing** [dapr/components-contrib#2249]
## Expected Behavior

send message with UTF8 characters (like Chinese) into azure storage queue, and can be processed by Azure Function QueueTrigger.

## Actual Behavior

 Azure Function QueueTrigger report this error:

```log
The input is not a valid Base-64 string as it contains a non-base 64 character, more than two padding characters, or an illegal character among the padding characters.
```

## Steps to Reproduce the Problem

- Create dapr app with output binding (use bindings.azure.storagequeues)
- Send a model with daprClient.InvokeBindingAsync("binding-output-importuser", "create", input);
- Create an Azure Function QueueTrigger with same Azure Storage Account to receive the message from output binding

## Release Note

Dapr runtime: 1.8.4

**Comment by berndverst** [comment]
This was broken in the following PR: https://github.com/dapr/components-contrib/pull/1156

Before that Chinese and other UTF8 characters should have worked fine.

**Comment by berndverst** [comment]
I opened a PR to add base64 support and added a test case which includes Chinese characters and show this is working correctly. This will be released in Dapr 1.10.

**Comment by berndverst** [comment]
As a temporary workaround you will need to perform the base64 encoding of your data in your application code at this time.
